makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: clean linux windows
+
+# Create a python environment and install dependencies This will create a conda
+# environment in the ./env directory and install the necessary packages for
+# FLFM.
+env/bin/python:
+	@echo "Creating conda environment..."
+	conda create -y --prefix ./env python=3.11
+	conda run -p ./env python -m pip install git+https://github.com/ssec-jhu/flfm.git
+
+models/model1.pt: env/bin/python
+	@echo "Exporting models..."
+	mkdir -p models
+	@for i in $(shell seq 1 15); do \
+		echo "Exporting model$$i"; \
+		conda run -p ./env python -m flfm.cli export --out models/model$$i.pt --n_steps $$i; \
+	done
+
+flfm-ij/src/main/resources/models/model1.pt: models/model1.pt
+	@echo "Copying model1.pt to flfm-ij/src/main/resources/models"
```

Comment:
These static refs to ``model1.pt`` don't look correct. At least the printed messages could say, ``Copying model*.pt`` or ``Copying models``, right? Or perhaps I've misunderstood the ref to ``model1``.

Reply:
You're right, thanks. I'll update.